### DISCUSSION TITLE
VSCODE github copilot plugin don`t work.

### DISCUSSION THREAD

**Discussion by Bajdzis** [community/community#52750] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi, I have a problem with github copilot on my Windows. (On my other PC with linux works fine).

It seems to be some kind of problem with running the plugin. The category for the copilot plugin does not appear in the output tab. However, in the developers tool I see the following error. 

```
Activating extension 'GitHub.copilot' failed: Cannot find module '../bin/napi-v3/win32/ia32/onnxruntime_binding.node'
Require stack:
- c:\Users\Bajdzis\.vscode\extensions\github.copilot-1.79.10634\dist\node_modules\onnxruntime-node\dist\binding.js
- c:\Users\Bajdzis\.vscode\extensions\github.copilot-1.79.10634\dist\node_modules\onnxruntime-node\dist\backend.js
- c:\Users\Bajdzis\.vscode\extensions\github.copilot-1.79.10634\dist\node_modules\onnxruntime-node\dist\index.js
- c:\Users\Bajdzis\.vscode\extensions\github.copilot-1.79.10634\dist\extension.js
- f:\Microsoft VS Code\resources\app\out\vs\loader.js
- f:\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- f:\Microsoft VS Code\resources\app\out\bootstrap-fork.js
```

My vscode version: 
```
Version: 1.77.3 (system setup)
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T10:34:05.691Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT ia32 10.0.19045
Sandboxed: No
```

**Comment by sahana-github** [comment]
@Bajdzis It looks like there is an issue with the installation of the GitHub Copilot extension on your Windows machine. Specifically, the error message indicates that the onnxruntime_binding.node module cannot be found.

**Reply by Bajdzis** [reply]
The only problem is how to fix it? I updated VScode, uninstalled and installed the plugin. tried other versions plugin (1.79.10634, 1.82.15, nightly-1.82.22, nightly-1.82.21). And all the time I have the same error. The strangest thing is that for the first few days it worked fine :/. 

How to debug my problem?

**Reply by darthwalsh** [reply]
@Bajdzis  try v1.73.8685 -- I had the same issue as you, but going back versions a couple months seems to have solved it for now.

**Comment by sahana-github** [comment]
@Bajdzis   You can try to get more information about the error by checking the extension logs

**Comment by darthwalsh** [comment, marked as answer]
I have the same issue. Looks to be caused by microsoft/onnxruntime#15486

I have 32-bit VSCODE on my PC, which increasingly seems like a less-supported option. I've been dragging my heels about uninstalling and switching to 64-bit, but maybe this is the final straw.

---

Errors says

    Error: Cannot find module '../bin/napi-v3/win32/ia32/onnxruntime_binding.node

But opening that folder, the `ia32` folder doesn't exist:

```
C:\Users\cwalsh\.vscode\extensions\github.copilot-1.82.15\dist\node_modules\onnxruntime-node\bin\napi-v3\win32\arm64
C:\Users\cwalsh\.vscode\extensions\github.copilot-1.82.15\dist\node_modules\onnxruntime-node\bin\napi-v3\win32\x64
```

**Reply by Bajdzis** [reply]
That was a good idea I was using the 32 bit version. After migrating to the 64 bit version it started working!

**Comment by hwleungcsk** [comment]
Are you using ARM machine?
I got the same problem with surface pro x

a stupid workaround: install old version of copilot, my vs code arm64 can work with copilot now.

Mac: v1.82.15 works in VS Code
Windows 10: v1.82.15 works in VS Code
Windows 11 ARM in Parallels: only v1.78.9758 works
https://github.com/orgs/community/discussions/51923

**Reply by Bajdzis** [reply]
I have been using VSCode since the alpha version. And I only had the 32 bit version installed. After migration to 64 bit version in me it started to work.
